salt/jupyter/files/data_generator.py

```diff
@@ -43,27 +45,23 @@ def generate_sample_datasets (host_ips, metric_ids, year, month, day, hour):
         for host_ip in host_ips:
            record = {}
            record['timestamp'] = (ts * 1000)
-           record['src'] = 'test'
-           record['host_ip'] = host_ip
-           record['rawdata'] = generate_random_metrics(metric_ids)
+           record['source'] = 'samples'
+           record['rawdata'] = generate_random_metrics(host_ip, metric_ids)
            #encode avro
            writer.append(record)
     writer.close()
-    subprocess.Popen(['sudo', '-u', 'hdfs', 'hadoop', 'fs', '-copyFromLocal', tmp_file, dir])
+    subprocess.Popen(['hadoop', 'fs', '-copyFromLocal', tmp_file, dir])
```

Comment:
This change is good but means this now needs to be run as a user that has write permissions to the relevant directory, rather than a sudoer. Is that documented?